team-telnyx/webrtc
# feat(WEBRTC-445): update vulnerability dependency

- Updated vulnerability issues to close some issues on Github WhiteSource.

## 📝 To Do


## ✋ Manual testing

1. Run `npm i` or `yarn` on root folder
2. Navigate to `packages/js` and run `npm i && npm run build && npm run test` See if they are all succeed.
3. Navigate to `packages/js/examples/react-audio-widgets` and run `npm run setup && npm run storybook`
4. Choose one of the examples on storybook and make a call.

## 🦊 Browser testing

### Desktop


## 📸 Screenshots

| Description | Screenshot |
| ----------- | ---------- |
| Desktop     |            |
| usage.gif   |            |
